### DISCUSSION TITLE
v5.1.0

### DISCUSSION THREAD

**Discussion by harshankur** [harshankur/officeParser#49] [Announcements]
# v5.1.0 - 12.11.2024
## What's Changed
* Added support for ArrayBuffer as file input.
* Fixed #35 which was a request to create browser version by creating bundle.js files available as part of release asset which exposes officeParser as a namespace.

<hr /><em>This discussion was created from the release <a href='https://github.com/harshankur/officeParser/releases/tag/v5.1.0'>v5.1.0</a>.</em>

**Comment by gladishd** [comment]
Release v5.1.0 · harshankur/officeParser
https://github.com/harshankur/officeParser/releases/tag/v5.1.0
Thanks for the `officeParserBundle@5.1.0.js`, however we additionally need the worker file `officeParserBundle@5.1.0.worker.js`! The following will generate the worker file `officeParserBundle@5.1.0.worker.js` : 
`npm install`
`npx browserify officeParser.js --standalone officeParser -o dist/officeParserBundle@5.1.1.js`
In `pdfjs-dist-build`, rename `pdf.worker.js` to `officeParserBundle@5.1.0.worker.js`

**Reply by harshankur** [reply]
I am sorry. I have uploaded it in the [release note](https://github.com/harshankur/officeParser/releases/tag/v5.1.0).
